main.tf

```diff
@@ -9,6 +9,12 @@ resource "aws_instance" "webserver" {
   tags = {
     Name = "webserver"
   }
+
+  metadata_options {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure at-rest server side encryption (SSE) is enabled for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets do not have at-rest server side encryption enabled to protect sensitive data.
 
**Remediation**
On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS).

source: accurics-stageone